internal/cli/root/atlas/builder.go

```diff
@@ -124,7 +124,7 @@ func Builder(profile *string) *cobra.Command {
 			if check, isHb := notifier.shouldCheck(); check {
 				_ = notifier.notifyIfApplicable(isHb)
 			}
-			telemetry.TrackCommand(cmd)
+			telemetry.TrackCommand(cmd, nil)
```

Comment:
[q] Why in atlascli the error is always `nill`?

Reply:
Success calls won't have an error only, originally I had TrackCommand and TrackCommandError.  As @johansteyn pointed they were nearly identical so he suggest to merge the functions.